xbmc/utils/LangCodeExpander.cpp

```diff
@@ -410,6 +413,24 @@ bool CLangCodeExpander::LookupInUserMap(const std::string& code, std::string& de
   return false;
 }
 
+bool CLangCodeExpander::LookupInLangAddons(const std::string& code, std::string& desc)
+{
+  if (code.empty())
+    return false;
+
+  std::string sCode(code);
+  StringUtils::ToLower(sCode);
+  StringUtils::Trim(sCode);
```

Comment:
Nitpicking, but trim should be done first. There are chances that tolower has to deal with less characters. :-)

Reply:
nice right